docs/modules/operation/pages/geographical-map/searching.adoc

```diff
@@ -0,0 +1,73 @@
+## Searching
+
+The search bar has the ability to do a simple text search, as well as searching particular fields associated with the node.
+
+### Search Fields
+
+The following fields can be searched:
+
+[options="header"]
+[cols="2*"]
+|====
+| Field
+| Description
+
+| `nodeId`
+| The ID of the node.
+
+| `nodeLabel`
+| A text description.
+
+| `foreignSource`
+| The provisioned foreign source, if any.
+
+| `foreignId`
+| The provisioned foreign ID, if any.
+
+| `ipAddress`
+| The primary IP address.
+
+| `severityLabel`
+| The severity. ("Normal", "Warning", etc.)
+
+| `description`
+| The description field from the node's assets.
+
+| `maintContract`
+| The maintenance contract from the node's assets.
+
+| `category`
+| Match a category, if any.
+
+|====
+
+Note that *all* searches are done in a case-insensitive manner.
+
+### Substring Matching Example
+
+To search for text in one of the fields, use the following syntax:
+
+`fieldname:searchtext`
+
+If the search is `nodelabel:server` it will match a node's label like `server1.opennms.com` or `server2.opennms.com`
+
+### Exact Matching Example
+
+To search for an exact match of a field, use the following syntax:
+
+`fieldname=searchtext`
+
+If the search is `nodelabel=server1.opennms.com` it will only match a node's label like `server1.opennms.com`.
```

Comment:
```suggestion
If the search is `nodelabel=server1.opennms.com` it will only match a node's label like `server1.opennms.com`.
```

Reply:
@Bonrob2 GH tells me here that there is no change. Did you change something?

Reply:
it's a whitespace cleanup. (extra space at the end of the line)
![image](https://user-images.githubusercontent.com/26419758/158863514-28f976ad-6ed2-45c1-8383-d064a9057048.png)

Reply:
Seems to be solved